apps/mobile-wallet/src/api/transactions.ts

```diff
@@ -1,17 +1,14 @@
-import { AddressHash, AssetAmount, client } from '@alephium/shared'
+import { AddressHash, AssetAmount, client, DEPRECATED_Address as Address } from '@alephium/shared'
```

Comment:
I moved the mobile wallet `Address` type to shared and I renamed it to `DEPRECATED_Address`. By importing it and renaming it with `DEPRECATED_Address as Address` makes it easy for migration since I'll just need to remove the `DEPRECATED_Address as ` part once migration is done.